docs/tutorial.md

```diff
@@ -188,19 +188,21 @@ The use of `baz` can be done in three ways; which way you choose is up to you an
 The first method is by using the `OverloadedLabels` extension and importing the orphan instance of `IsLabel` for the `Lens` type from `Data.ProtoLens.Labels`.
 That gives us the use of `#` for prefixing our field accessors.
 ``` haskell
-{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE OverloadedLabels  #-}
+{-# LANGUAGE OverloadedStrings #-}
 
+import Data.ProtoLens        (defMessage)
 import Data.ProtoLens.Labels ()
-import Microlens             ((^.))
-import Proto.Foo          as P
+import Lens.Micro            ((&), (.~), (^.))
+import Proto.Foo             as P
 
 myBar :: P.Bar
 myBar = defMessage
             & #baz   .~ 42
             & #bippy .~ "querty"
 
 main :: IO ()
-main = putStrLn $ myBar ^. #bippy
+main = print $ myBar ^. #bippy
```

Comment:
Why is `print` preferable to `putStrLn` here?

Reply:
Because the argument is of type `Text`, we cannot use `putStrLn` from Prelude.
You could alternatively import `putStrLn` from Data.Text.IO, but I thought it'd just suffice to use `print` as the other examples use it too.

Reply:
Makes sense. Thanks for the explanation.